src/functions/helper_lookup.ts

```diff
@@ -62,8 +63,7 @@ export function addPivotDependencies(
 
   for (const measure of forMeasures) {
     if (measure.computedBy) {
-      const formula = evalContext.getters.getMeasureCompiledFormula(measure);
-      dependencies.push(...formula.dependencies.filter((range) => !range.invalidXc));
+      dependencies.push(...evalContext.getters.getMeasureFullDependencies(pivotId, measure));
```

Comment:
Since the new approach where we get all dependencies, this adds the same dependencies multiple times.
```ts
dependencies.push(...evalContext.getters.getMeasureFullDependencies(pivotId));
```